ibis_substrait/compiler/decompile.py

```diff
@@ -730,7 +731,7 @@ def _decompile_expression_scalar_function(
 ) -> ir.ValueExpr:
     extension = decompiler.function_extensions[msg.function_reference]
     function_name = extension.name
-    op_type = getattr(ops, inflection.camelize(function_name))
```

Comment:
Can we remove the `inflection` dependency in this PR?

Reply:
done